precompiles/utils/macro/tests/precompile/expand/returns_tuple.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019-2022 PureStake Inc.
```

Comment:
2023? :eyes:

Reply:
Seems we didn't do the "Happy New Year" PR yet :p